src/cloud-api-adaptor/test/tools/provisioner-cli/main.go

```diff
@@ -36,6 +37,7 @@ func main() {
 	cloudProvider := os.Getenv("CLOUD_PROVIDER")
 	provisionPropsFile := os.Getenv("TEST_PROVISION_FILE")
 	podvmImage := os.Getenv("TEST_PODVM_IMAGE")
+	shouldDeployKbs := os.Getenv("DEPLOY_KBS")
```

Comment:
```
shouldDeployKbs := false
if os.Getenv("DEPLOY_KBS") == "true" || os.Getenv("DEPLOY_KBS") == "yes" {
   shouldDeployKbs = true
 }
```
 Can we do it like this ? So that later on we can just check for  shouldDeployKbs  without using `strings.EqualFold` but just bool condition and multiple checks can be avoided.